src/foam/java/refinements.js

```diff
@@ -294,16 +298,18 @@ foam.LIB({
 
       var axioms = this.getOwnAxioms();
 
-      for ( var i = 0 ; i < axioms.length ; i++ ) {
+      for ( var i = 0; i < axioms.length; i ++ ) {
```

Comment:
space before ;'s, no space before ++.

Reply:
Fixed for all `for`s: https://github.com/foam-framework/foam2/pull/1239